### ISSUE TITLE
[9.3.*] Dynamic Planning view vs List planning

### ISSUE THREAD

**Issue by M-Falken** [glpi-project/glpi#4612]
Hi
when we use a phone, the planning view (with dynamic tabs) is practically unusable on the weekly and monthly view as they are overwritten because there is no horizontal slider.
The list view available through the tab "My schedule" as well as through the home page is much more convenient according to our techs, however it does not have the essential information unlike the dynamic schedule.
Tooltip information does not help for phone users.

So It is missing in the list view:

- The name of the day next to the date (the date is not enough for the technicians when they want to see a preview of their week)
- The entity
- The equipment concerned
- The place

To put it simply, what is displayed in the tooltip should also appear just under the title, in the list view.
Thanks a lot to consider.
![screenshot_20180911-105150](https://user-images.githubusercontent.com/1380639/45349397-70bbe100-b5b1-11e8-8ebb-d2b10fc9f280.jpg)

```
GLPI 9.3 prod
GLPI 9.3.1test
```

**Comment by M-Falken** [comment]
Same with latest build.
A simple slider on the bottom would be a good start.

**Comment by M-Falken** [comment]
Nothing new ?
